src/main/java/school/lemon/changerequest/java/introduction/hw2/Task1.java

```diff
@@ -1,7 +1,8 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
 public class Task1 {
-    public static String makeTags(String tag, String text) {
-        return "";
+    public static String makeTags(String tag, String text)
+    {
+        return String.format("%1$s%2$s%3$s%4$s%5$s%2$s%3$s", "<", tag, ">", text, "</", tag, ">");
```

Comment:
You can print any constant values in pattern, for example:
`String.format("<%1$s>", "tag")` will print `"<tag>"`.